lib/Driver/ToolChains.cpp

```diff
@@ -1044,6 +1044,24 @@ addLinkSanitizerLibArgsForDarwin(const ArgList &Args,
                      /*AddRPath*/ true, TC);
 }
 
+static void
+addLinkFuzzerLibArgsForDarwin(const ArgList &Args,
+                       ArgStringList &Arguments,
+                       const ToolChain &TC) {
+
+  // libFuzzer requires C++.
+  Arguments.push_back("-lc++");
+
+  // Link libfuzzer.
+  SmallString<128> Dir;
+  getRuntimeLibraryPath(Dir, Args, TC);
+  llvm::sys::path::remove_filename(Dir);
+  llvm::sys::path::append(Dir, "llvm", "libLLVMFuzzer.a");
```

Comment:
I'm confused about this. There's no such thing as an "LLVM resource dir" in a normal installation of Clang. Where does libLLVMFuzzer.a live in that case?

(Also, http://llvm.org/docs/LibFuzzer.html talks about "libFuzzer.a" rather than "libLLVMFuzzer.a". Are they equivalent?)